### PR TITLE
Add Xtremepush (Actions) Destination docs.

### PR DESCRIPTION
**Pull request by vii07**
### Proposed changes
Add the new docs page for Xtremepush (Actions) Destination.

### Merge timing
ASAP once approved.

### CODE REVIEW

File: src/connections/destinations/catalog/actions-xtremepush/index.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+Some special traits will also be used as additional user identifiers:
+
+| Segment Trait | Xtremepush User Identifier |
+| ------------- | -------------------------- |
+| email         | email                      |
+| phone         | mobile_number              |
+
+For any additional traits you want to save you should create [User Profile Attributes](https://docs.xtremepush.com/docs/attributes-tags){:target="_blank"} in your Xtremepush Project.
+
+If a trait does not match a custom Xtremepush User Profile Attribute and is not recognized as a User Identifier it will be ignored.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-xtremepush/index.md:53]:
```suggestion
If a trait does not match a custom Xtremepush User Profile Attribute and is not recognized as a User Identifier, Xtremepush ignores the trait.
```

File: src/connections/destinations/catalog/actions-xtremepush/index.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+If you're not familiar with the Segment Specs, take a look to understand what the [Track method](/docs/connections/spec/track/) does. An example call would look like:
+
+```
+analytics.track('Product Purchased', {
+  productName: 'Some Product'
+})
+```
+
+Track calls will be sent to Xtremepush as a `event hits`, so you can use it to [trigger a campaign](https://docs.xtremepush.com/docs/campaign-events){:target="_blank"} for a user.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-xtremepush/index.md:65]:
```suggestion
Track calls are sent to Xtremepush as a `event hits` and you can use them to [trigger a campaign](https://docs.xtremepush.com/docs/campaign-events){:target="_blank"} for a user.
```

File: src/connections/destinations/catalog/actions-xtremepush/index.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+Event properties can be used as merge tags in the message content. You can also define additional rules on where to trigger the campaign based on event properties value.
+
+## Enabling Push and In-App Notifications
+To enable Xtremepush push and in-app notifications you will also need to to install the relevant Xtremepush SDKs.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-xtremepush/index.md:70]:
```suggestion
To enable Xtremepush push and in-app notifications you must also install the relevant Xtremepush SDKs.
```

**Review by forstisabella — CHANGES_REQUESTED** [review]

File: src/connections/destinations/catalog/actions-xtremepush/index.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
+---
+title: Xtremepush (Actions) Destination
+---
```
[the remark this answers is not in the document]
Comment (vii07) [reply src/connections/destinations/catalog/actions-xtremepush/index.md:3]: @forstisabella Thanks for your suggestions and I'll apply all of them soon. But a note for this one is that we want to make our new Xtremepush (Actions) Destination public beta after merging this doc, so I will pass `hidden: false` instead.

**Review by tcgilbert — APPROVED** [review]
lgtm

File: src/connections/destinations/catalog/actions-xtremepush/index.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+title: Xtremepush (Actions) Destination
+beta: true
+hidden: false
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-xtremepush/index.md:4]:
```suggestion
```

**Review by forstisabella — APPROVED** [review]

File: src/connections/destinations/catalog/actions-xtremepush/index.md (hunk 5 above)
Comment (forstisabella) [reply src/connections/destinations/catalog/actions-xtremepush/index.md:4]: we can remove this attribute if hidden is not set to true, as it's not hidden by default!

**Pull request merged**